.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CRYPTOWATCHER
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Lint code
+        run: yarn lint
+
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build code
+        run: yarn build:ts
+
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install SQLite
+        run: sudo apt-get install sqlite3 libsqlite3-dev
+  
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+  
+      - name: Create SQLite database file
+        run: touch ./test/test.db
+  
+      - name: Set environment variables for test
+        run: echo "DATABASE_URL=./test/test.db" >> $GITHUB_ENV
+  
+      - name: Run tests with SQLite
+        run: yarn test
+  
+      - name: Delete SQLite database
+        run: rm ./test/test.db
```

Comment:
Do this setup in `test` command in `package.json`. Here you just have to call `yarn test` same as you did for `yarn build`.

Reply:
I changed it.